### ISSUE TITLE
underline and double-underline accessibility 

### ISSUE THREAD

**Issue by lwtchu** [openstax/recipes#137]
sister issue: https://github.com/openstax/cnx-recipes/issues/4103

make underline/double-underline accessible with the way discussed [in accessibility channel here](https://openstax.slack.com/archives/C4PJYP0LF/p1634583632041700) and  with the unified-content meeting - [notes under oct 21 ](https://docs.google.com/document/d/1fgx3zfprxDi-yg372KqMQQ8QGZhGbVQpl4JfDGqRuTE/edit#heading=h.1iqin8y6n49a)

content: github book osbooks-writing-guide

**Comment by lwtchu** [comment]
if this is for all books would it be in kitchen 🤔

**Comment by lwtchu** [comment]
1. use recipe to add the
```<span data-screenreader="true">
            double-underline
        </span>```
and
```<span data-screenreader="true">
            underline
        </span>```
to the content 

2. suppress with css https://zellwk.com/blog/hide-content-accessibly/

**Comment by kswanson33** [comment]
attribute should be `data-screenreader-only="true"` as per phil

**Comment by Stackblocks** [comment]
@philschatz @kswanson33 

## The Good News
The span is being properly inserted and hidden

![image](https://user-images.githubusercontent.com/12678235/141865217-38e7df08-6938-43d8-92fb-68e423dd51ab.png)

## The Bad News
JAWS isn't announcing either the standard underline or the hidden double-underline span

https://user-images.githubusercontent.com/12678235/141865231-0e392ae8-e4b5-4228-ae62-2c88349bfbcb.MOV

## Next Steps
We may need to try out a different solution for this, but before we do that I wanted to check and see if anyone knew someone who was more of an expert at screen reader use. 

It may well be that I am using the screenreader with settings or commands that, while recommended by the documentation, are not a match for how a native user would use the screenreader and I want to try and rule that out before we go down another path if possible.

**Comment by Stackblocks** [comment]
Here is a copy of the [accessible underlines test data](https://github.com/openstax/recipes/files/7542110/english_underlines.zip) I was running through the screenreader in case someone else wants to take a look.

**Comment by lwtchu** [comment]
Let's roll back the changes for now until we find another alternative for screenreaders.

**Comment by philschatz** [comment]
From the attached CSS file it seems the screenreader-only text was hidden using the `display: none;` technique which hides content from the screen _and_ from screenreaders.

```
span[data-screenreader-only="true"] {
  display: none; }
```

Fortunately, it can be replaced with another technique: [Slack](https://openstax.slack.com/archives/C4PJYP0LF/p1636388726005000?thread_ts=1634583632.041700&cid=C4PJYP0LF)

See https://zellwk.com/blog/hide-content-accessibly/

**Comment by kswanson33** [comment]
oh no my reading comprehension :facepalm:

**Comment by Stackblocks** [comment]
**_Caution: Visible flashing / screen strobe due to using a camera to film a display._**

"Underline" and "Double Underline" are being read out successfully by the screenreader.

https://images.zenhubusercontent.com/321826244/b8af66b9-f072-4176-8a31-6ec61d108500/dowble_underlinnay.mov

**Comment by lwtchu** [comment]
![image](https://user-images.githubusercontent.com/11299125/148413883-27074207-e44c-417f-8533-8adc83525c1d.png)

beautiful accessibility words on rex
